workers/api-server/src/http/wallet.rs

```diff
@@ -409,30 +454,9 @@ impl TypedHandler for CreateOrderHandler {
         params: UrlParams,
         _query_params: QueryParams,
     ) -> Result<Self::Response, ApiServerError> {
-        let id = req.order.id;
         let wallet_id = parse_wallet_id_from_params(&params)?;
 
-        // Lookup the wallet in the global state
-        let old_wallet = find_wallet_for_update(wallet_id, &self.state).await?;
-        let mut new_wallet = old_wallet.clone();
-        let new_order: Order = req.order.into();
-
-        // Check that the timestamp is not too old, then add to the wallet
-        new_wallet.add_order(id, new_order.clone()).map_err(bad_request)?;
-        new_wallet.reblind_wallet();
-
-        let task = UpdateWalletTaskDescriptor::new_order(
-            new_order,
-            id,
-            old_wallet,
-            new_wallet,
-            req.statement_sig,
-        )
-        .map_err(bad_request)?;
-
-        // Propose the task and await for it to be enqueued
-        let task_id = append_task_and_await(task.into(), &self.state).await?;
-        Ok(CreateOrderResponse { id, task_id })
+        create_order(req.order, req.statement_sig, wallet_id, &self.state, None).await
```

Comment:
nit: add `/* pool_name */` after the `None`; good to avoid literals without documentation